options/options.js

```diff
@@ -1,9 +1,16 @@
+"use strict";
+// Defaults
+var DecoderMemoryLimitInMegabytes = 32;
```

Comment:
Use `const` instead of `var` for these and prefix both names with `Default` to clearly show later in the code that it is the default value we are using.